src/graphql/query/categoryRows.graphql

```diff
@@ -13,5 +13,13 @@ query categoryRows {
 				}
 			}
 		}
+		countryFacets {
```

Comment:
Adding this was the only way I could resolve an invariant error. 😕

Reply:
This is an expensive thing to be adding to a query that doesn't appear to be using it. Where were you getting an invariant violation?

Reply:
On the homepage.